frontend/src/components/emailConfirmation.js

```diff
@@ -0,0 +1,9 @@
+const EmailConfirmation = () => {
+  return (
+    <div>
+      <h1>Moderator: I send and email to confirm user's email on this page</h1>
```

Comment:
- can you reprhase the message to more understable. Or you can check sample messages from google.

Reply:
It is not really part of the project, it is just a note to inform that I send an confirmation email on that page, I've actually just completed that page, should I still change it?, please let me know